Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -269,14 +277,14 @@ class FavoritesViewController: UIViewController, Themeable {
         
         let adsAvailableInRegion = BraveAds.isCurrentLocaleSupported()
         
-        // Don't show notifications if background images are disabled.
-        guard let bgImageInfo = backgroundImage.info else { return nil }
+        if !Preferences.NewTabPage.backgroundImages.value { return nil }
         
+        let isSponsoredImage = background?.sponsor != nil
```

Comment:
@jhreis pls confirm if this is enough to determine image sponsorship

Reply:
It is enough to determine if the _current_ image is a sponsored image.